cs-utilities/src/main/java/io/cloudslang/content/utilities/services/osdetector/OperatingSystemDetector.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (c) Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+*/
+package io.cloudslang.content.utilities.services.osdetector;
+
+import io.cloudslang.content.utilities.entities.OperatingSystemDetails;
+import io.cloudslang.content.utilities.entities.OsDetectorInputs;
+import io.cloudslang.content.utilities.util.OsDetectorUtils;
+
+public class OperatingSystemDetector {
```

Comment:
Wouldn't be more appropriate to name this "OperatingSystemDetectorService" and the interface "OperatingSystemDetector" or simply "OsDetector"?